### PR TITLE
executor: remove Next function for RevokeExec

### PR DESCRIPTION
**Pull request by yangwenmai**

### CODE REVIEW

File: executor/revoke.go [hunk 1]
```diff
@@ -50,15 +50,6 @@ type RevokeExec struct {
...
-func (e *RevokeExec) Next(ctx context.Context) (Row, error) {
-	if e.done {
-		return nil, nil
-	}
-	e.done = true
-	return nil, errors.Trace(e.run(ctx))
-}
-
 // NextChunk implements the Executor NextChunk interface.
 func (e *RevokeExec) NextChunk(ctx context.Context, chk *chunk.Chunk) error {
```
Comment (zz-jason) [inline executor/revoke.go]: `e.run` can be removed and put the codes into `NextChunk`

**Comment by XuHuaiyu** [comment]
Please address comment. @yangwenmai

File: executor/revoke.go (hunk 1 above)
Comment (yangwenmai) [reply executor/revoke.go]:
```go
    if e.done {
        return nil
    }
    err := e.run(ctx)
    e.done = true
    return errors.Trace(err)
```
Is it?

**Pull request merged**